### DISCUSSION TITLE
Different roles to adjust users risght

### DISCUSSION THREAD

**Discussion by lkino** [opengisch/QField#2626] [Q&A]
It seem, there are some roles (like: inspector, admin, editor, reader....), it is certainly to adjust users right into the application but we doesn't find any description of these roles. 
We can find also on each change line there is a box whitch one is disable... Is it for approve or reject each change? That can be very interresting to have double check and to avoid to integrate error from field engineer?
Best regards

**Comment by suricactus** [comment]
We are still working on the docs, but you can see at least an intro here: https://docs.qfield.org/reference/qfieldcloud/permissions/?h=#roles
If you have any suggestions on improvements, please let us know.

> We can find also on each change line there is a box whitch one is disable... Is it for approve or reject each change? 

Your are right, this is an action to be applied if the change has a FAILED status, which means the change failed to be applied on the original dataset. You see the box as disabled because you cannot do much with your current changes. If the change has a FAILED status you can choose to retry or reject the change.

> That can be very interresting to have double check and to avoid to integrate error from field engineer?

This is a planned feature that will be released later this year. If you are interested in speeding up the development, please contact us directly via email.
